lib/hubspotter/contact.rb

```diff
@@ -51,6 +51,11 @@ def self.update(vid, properties = {} )
       create_or_update(path, properties)
     end
 
+    def self.update_without_vid(properties = {})
```

Comment:
Rather than `update_without_vid` something like `update_by_email` seems more appropriate.

Since `email` is the primary lookup key and required for a successful request, I'd include it in the method signature as the first variable rather than pulling it from `properties`.  As it is now, if `:email` isn't defined in `properties` you'll get no error from Hubspotter, but rather you'll submit a bad request to the API and get an error back from Hubspot.